cypress/support/flightSearch/flightSearchCommands.cy.js

```diff
@@ -0,0 +1,29 @@
+Cypress.Commands.add("selectDeparture", () => {
+    cy.get('#input-button__departure').click()
+    
+    cy.get(':nth-child(2) > :nth-child(1) > .countries__country-inner').click()
```

Comment:
for example, here it's hard for me to figure out what exactly you're clicking? that's why we wrap all these element in POMs, for better understadning so these elements are not directly a part of test implementation. Remember, tests often serve as documentation! :)